src/ResourceManager/Sql/Commands.Sql/Database/Model/AzureSqlDatabaseModel.cs

```diff
@@ -206,26 +206,26 @@ public AzureSqlDatabaseModel(string resourceGroup, string serverName, Management
             CollationName = database.Collation;
             CreationDate = database.CreationDate.Value;
             CurrentServiceObjectiveName = database.ServiceLevelObjective;
-            MaxSizeBytes = long.Parse(database.MaxSizeBytes);
+            MaxSizeBytes = database.MaxSizeBytes.Value;
             DatabaseName = database.Name;
             Status = database.Status;
             Tags = TagsConversionHelper.CreateTagDictionary(TagsConversionHelper.CreateTagHashtable(database.Tags), false);
             ElasticPoolName = database.ElasticPoolName;
             Location = database.Location;
             ResourceId = database.Id;
             CreateMode = database.CreateMode;
-            EarliestRestoreDate = database.EarliestRestoreDate;
+            EarliestRestoreDate = null; // database.EarliestRestoreDate;
```

Comment:
Why are you making this change?

Reply:
This has to do with the changes that @jaredmoo is making where some of these things can't be set correctly until the new sdk is updated.

Reply:
So will this be reverted before you are ready to merge the PR?

Reply:
Correct. There are actually even more changes having to do with this now.